### DISCUSSION TITLE
Request: Add relevant tool bar buttons to the DAX Query window when it is a separate window

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1126] [Ideas]
It would be nice if the relevant buttons from the main window were available in the DAX query window when it is separated. Comment, Uncomment, Format, etc. The only buttons available are below. I could not figure out how to add them to the toolbar, and unfortunately, for some reason, the shortcut keys aren't working. 

![image](https://user-images.githubusercontent.com/45298358/218162996-e2176765-0c08-47fe-8460-6a811bccf14e.png)

For example, CTRL-SHIFT-K doesn't comment code, it just adds some unprintable ASCII format code (vt) and then the intellisense kicks in. I don't know if that is a bug in and of itself, or if it is the work laptop I am using, which may have some other software running on it that has its own set of shortcut keys overriding TE/3's shortcuts. Not my laptop, and I am not an admin of it.

![image](https://user-images.githubusercontent.com/45298358/218163661-68c050eb-15bb-4819-9bd3-442dccb91cd3.png)

**Comment by mlonsk** [comment]
Hi Ed (@edhans )
Thank you for the suggestion regarding the tool bars, I have passed it on to our development team. 

I have also been able to reproduce your shortcut issue, so it is not just your laptop. We will look into to it further. 
A small comment, the shortcut for commenting code is per default CTRL-K, it still doesn't work in the popped out window however.

We really appreciate you reporting these things to us.

**Comment by mlonsk** [comment]
Small update on the bug part:
It only seems to happen when either the Pivot Grid or DAX Query window is open in the main application.

**Comment by edhans** [comment]
Thanks. Per https://docs.tabulareditor.com/te3/features/shortcuts.html the shortcut for commenting is CTRL-SHIFT-K which is where I got that.

**Comment by mlonsk** [comment]
Hi @edhans 
Thank you for making us aware of that discrepancy. We will fix the documentation.
